packages/elements/src/clock/__test__/clock.interactive.test.js

```diff
@@ -18,12 +18,24 @@ describe('clock/Interactive', () => {
     }
 
     const InputKey = {
+      arrowLeft: {
+        ieKey: 'Left',
+        key: 'ArrowLeft',
+        which: 37,
```

Comment:
`which` and `keyCode` are deprecated APIs. You can safely remove those

Reply:
Updated to remove